NDB.Covid19/NDB.Covid19.iOS/Views/DailyNumbers/DailyNumbersViewController.cs

```diff
@@ -134,6 +134,9 @@ private void SetStyling()
 			ConfirmedCases_StackView.ShouldGroupAccessibilityChildren = true;
 			NumberOfTests_StackView.ShouldGroupAccessibilityChildren = true;
 			PatientsAdmitted_StackView.ShouldGroupAccessibilityChildren = true;
+			IntensiveCare_StackView.ShouldGroupAccessibilityChildren = true;
```

Comment:
Not sure which is this one, it is not used anymore as I remember.

Reply:
OK, it is used, it just was never a bug since horizontally this is a last box. But yes, we need to have this.